src/darwin/Framework/CHIP/CHIPDeviceController.mm

```diff
@@ -217,8 +217,8 @@ - (BOOL)startup:(_Nullable id<CHIPPersistentStorageDelegate>)storageDelegate
 
         chip::MutableByteSpan icac;
 
-        errorCode = _operationalCredentialsDelegate->GenerateNOCChainAfterValidation(
-            _localDeviceId, 0, ephemeralKey.Pubkey(), rcac, icac, noc);
+        errorCode = _operationalCredentialsDelegate->GenerateNOCChainAfterValidation(_localDeviceId, 0,
+            { { chip::kTestControllerCAT, chip::kUndefinedCAT, chip::kUndefinedCAT } }, ephemeralKey.Pubkey(), rcac, icac, noc);
```

Comment:
No, wait, why would we do that, exactly?  This is not test code; it should not be doing this.

Reply:
I've updated the implementation. Reading CATs Configuration from the persistent memory. However, the exact implementation of the mechanism is TODO - subject for another PR.